### DISCUSSION TITLE
How can I download a specific version of DependencyTracker Docker images

### DISCUSSION THREAD

**Discussion by mciccarone** [DependencyTrack/dependency-track#2370] [Q&A] [answered]
I need to download an older version of Dependency Track (4.6) ?

The instructions such as "curl -LO https://dependencytrack.org/docker-compose.yml" always grab the latest.

What additional parameters can I pass to retrieve a different version ?

Thanks

**Comment by syalioune** [comment]


**Reply by mciccarone** [reply]
Unfortunately the checked in docker-compose.yml files will always reference "latest" on the pull request.

**Comment by mciccarone** [comment, marked as answer]
- I figured it out, but it requires modification of the docker-compose.yml file.
- 
In particular lines, 14 and 99.

    image: dependencytrack/apiserver needs to be changed to dependencytrack/apiserver:<version>    (4.6.1)
    image: dependencytrack/frontend needs to be changed to dependencytrack/frontend:<version>   (4.6.3)

where <version> is one that is available from hub.docker.com.
